tests/nnstreamer_filter_extensions_common/unittest_tizen_template.cc

```diff
@@ -291,25 +303,50 @@ TEST (nnstreamer_EXT_NICK_NAME_basic_functions, invoke)
   ret = sp->open (&prop, &data);
   EXPECT_EQ (ret, 0);
 
-  ret = sp->getOutputDimension (&prop, &data, &res);
-  EXPECT_EQ (ret, 0);
-  output.size = gst_tensor_info_get_size (&res.info[0]);
-  output.type = res.info[0].type;
-  num_outputs = res.num_tensors;
+  ASSERT_TRUE ((sp->getInputDimension && sp->getOutputDimension) ||
+      (sp->setInputDimension));
 
-  ret = sp->getInputDimension (&prop, &data, &res);
-  EXPECT_EQ (ret, 0);
-  input.size = gst_tensor_info_get_size (&res.info[0]);
-  input.type = res.info[0].type;
-  num_inputs = res.num_tensors;
+  /** Decide the size for input and output */
+  if (sp->getInputDimension && sp->getOutputDimension) {
+    ret = sp->getOutputDimension (&prop, &data, &res);
+    EXPECT_EQ (ret, 0);
+    output.size = gst_tensor_info_get_size (&res.info[0]);
+    output.type = res.info[0].type;
+    num_outputs = res.num_tensors;
+    gst_tensors_info_free (&res);
+
+    ret = sp->getInputDimension (&prop, &data, &res);
+    EXPECT_EQ (ret, 0);
+    input.size = gst_tensor_info_get_size (&res.info[0]);
+    input.type = res.info[0].type;
+    num_inputs = res.num_tensors;
+    gst_tensors_info_free (&res);
+  } else {
+    GstTensorsInfo res;
```

Comment:
IMO, this line is unnecessary (duplicated line 287)

Reply:
Updated.